Snakes.py

```diff
@@ -8,15 +8,16 @@
 gameDisplay = pygame.display.set_mode((800,600))
 
 pygame.display.set_caption('SNAKES')
-img=pygame.image.load('Assets\Images\head.png')
+img=pygame.image.load('head.png')
 pygame.display.set_icon(img)
 dirn="right"
 clock = pygame.time.Clock()
 font= pygame.font.SysFont("comicsansms", 35)
 k=25
 def score(score):
-	t=font.render("Score: "+str(score), True, (0,0,0))
-	gameDisplay.blit(t,[0,0])
+    s = "            PRESS P TO PAUSE"
```

Comment:
This is not the right way. Remove the space before the message. and then render is properly using the pos passed as argument in the font.render function